assets/src/edit-story/components/helpCenter/toggle/test/index.js

```diff
@@ -17,11 +17,11 @@
  * External dependencies
  */
 import { fireEvent, screen } from '@testing-library/react';
+import { renderWithProviders } from '@web-stories-wp/design-system/src/testUtils';
```

Comment:
Note to self: good reminder to check how much sense it makes to share test utils like this

Reply:
There was no way around this. Including this in the root, broke many other tests.

Reply:
Yes I get that. Still something I'd like to look into as it might not be the best way to share test utils like this.

Reply:
I agree. Should this be in a different package? 

I would like to handle this in a different PR, this PR is massive as it is!

Reply:
It should not block this PR. It's just a note to myself to think about this.